samples/notebooks/dotnet/09-BingSearch-using-kernel-prince.ipynb

```diff
@@ -0,0 +1,320 @@
+    "\n",
+    "Check other pricing from [here](https://www.microsoft.com/en-us/bing/apis/pricing).\n",
+    "\n",
+    "To learn more read the follwing [documenation](https://learn.microsoft.com/en-us/bing/search-apis/bing-web-search/overview)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Prepare a semantic kernel instance first, loading also the AI backend settings defined in the [Setup notebook](0-AI-settings.ipynb):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#r \"nuget: Microsoft.SemanticKernel, 0.16.230615.1-preview\"\n",
+    "#r \"nuget: Microsoft.SemanticKernel.Skills.Web, 0.16.230615.1-preview\"\n",
+    "\n",
+    "#!import config/Settings.cs\n",
+    "#!import config/Utils.cs\n",
+    "\n",
+    "using Microsoft.SemanticKernel;\n",
+    "using Microsoft.SemanticKernel.CoreSkills;\n",
+    "using Microsoft.SemanticKernel.SkillDefinition;\n",
+    "using Microsoft.SemanticKernel.Orchestration;\n",
+    "using Microsoft.SemanticKernel.Planning;\n",
+    "using Microsoft.SemanticKernel.Planning.Sequential;\n",
+    "using Microsoft.SemanticKernel.TemplateEngine;\n",
+    "var builder = new KernelBuilder();\n",
+    "\n",
+    "// Configure AI backend used by the kernel\n",
+    "var (useAzureOpenAI, model, azureEndpoint, apiKey, orgId) = Settings.LoadFromFile();\n",
+    "\n",
+    "if (useAzureOpenAI)\n",
+    "    builder.WithAzureTextCompletionService(model, azureEndpoint, apiKey);\n",
+    "else\n",
+    "    builder.WithOpenAITextCompletionService(model, apiKey, orgId);\n",
+    "\n",
+    "var kernel = builder.Build();"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You requried the following namespaces to use bing search ."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "using Microsoft.SemanticKernel.Skills.Web;\n",
+    "using Microsoft.SemanticKernel.Skills.Web.Bing;"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "// Load Bing skill\n",
+    "//Uncomment the Example \n",
+    "using (var bingConnector = new BingConnector(\"17172014d4754369bc9a422b9bae1345\"))\n",
```

Comment:
Should prompt user to enter API key as we do in `0-AI-settings.ipynb` and retrieve key from configuration when needed.

Reply:
This Bing credential is only going to be used here , thus I don't think the need to add this in `0-AI-settings.ipynb` .